### PR TITLE
Miguelandrade4/20 server create user / login

### PR DESCRIPTION
**Pull request by miguelandrade4**
This is for the completion of issue #20. Login and User Creation works as intended, but may be subject to change. Unit tests have been pushed back so for now AuthenticatorControllerSpec is just a skeleton for unit tests that will come later.

### CODE REVIEW

File: Server/test/controllers/AuthenticatorControllerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import org.scalatestplus.play.PlaySpec
+import org.scalatestplus.play.guice.GuiceOneAppPerTest
+import play.api.mvc.Result
+import play.api.test.Helpers.contentAsString
+import play.api.test.{FakeRequest, Helpers, Injecting}
+
+import scala.concurrent.Future
+
+class AuthenticatorControllerSpec extends PlaySpec
```
Comment (SamuelLabrador) [inline Server/test/controllers/AuthenticatorControllerSpec.scala]: Can you comment out this class, it's causing the build to fail.

**Review by SamuelLabrador — COMMENTED** [review]
Changes look good. One comment about unit test

File: Server/test/controllers/AuthenticatorControllerSpec.scala (hunk 1 above)
Comment (miguelandrade4) [reply Server/test/controllers/AuthenticatorControllerSpec.scala]: Done

**Review by SamuelLabrador — APPROVED** [review]
Changes look good. Ready 2 merge

**Pull request merged**